ixmp/tests/test_cli.py

```diff
@@ -34,8 +34,8 @@ def test_versiontype() -> None:
         vt.convert("xx", None, None)
 
 
-# FIXME This should work for IXMP4Backend, I think
-@pytest.mark.jdbc
+# FIXME The URL needs to be different for ixmp4; can you target model/scenario on pg?
```

Comment:
The ``ixmp://`` URL scheme uses the platform name appearing in the user's ixmp/config.json as the first part ('host'). So in the case of the local test platform, this should be `ixmp4-local`.